serverless/src/edlProfile/__tests__/handler.test.js

```diff
@@ -0,0 +1,65 @@
+import edlProfile from '../handler'
+
+import fetchEdlProfile from '../../../../static/src/js/utils/fetchEdlProfile'
+
+jest.mock('../../../../static/src/js/utils/fetchEdlProfile')
+
+beforeEach(() => {
+  jest.clearAllMocks()
+})
+
+describe('edlProfile when edl password is provided', () => {
+  test('when first, last name is given returns a 200 status code with the full name', async () => {
+    fetchEdlProfile.mockImplementation(() => ({
+      email: 'christopher.d.gokey@nasa.gov',
```

Comment:
is this something you mean to do?

Reply:
Would be great if we could mock this without the personal info :)